src/main/controllers/still_working/stillWorkingController.ts

```diff
@@ -0,0 +1,38 @@
+import { Response } from 'express';
+
+import { Form } from '../../components/form/form';
+import { AppRequest } from '../../definitions/appRequest';
+import { FormContent, FormFields } from '../../definitions/form';
+import { assignFormData, getPageContent, handleSessionErrors, setUserCase } from '../helpers';
+export default class stillWorkingController {
+  private readonly form: Form;
+
+  constructor(private readonly stillWorkingContent: FormContent) {
+    this.form = new Form(<FormFields>this.stillWorkingContent.fields);
+  }
+
+  public post = (req: AppRequest, res: Response): void => {
+    setUserCase(req, this.form);
+    let redirectUrl = '/are-you-still-working';
+    if (req.body.isStillWorking === 1) {
+      // this redirect URL below will need updated to the still working page
```

Comment:
Can you please add more meaningful comments with TODO's? So we know which pages this suppose to redirect to.

Reply:
updated to TODOs with specific page names